### PR TITLE
Databases week3

### PR DESCRIPTION
**Pull request by andrii1**

### CODE REVIEW

File: databases/week3/queries.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+CREATE TABLE `reservation` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `number_of_guests` int(10) unsigned NOT NULL,
+  `meal_id` int(10) unsigned NOT NULL,
+  `created_date` DATE NOT NULL,
+  `contact_phonenumber` varchar(255) NOT NULL,
+  `contact_name` varchar(255) NOT NULL,
+  `contact_email` varchar(255) NOT NULL,
+  CONSTRAINT `fk_consultations_meal` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
```
Comment (Elia-Git) [inline databases/week3/queries.sql]: good

File: databases/week3/additional_queries.sql [hunk 2]
```diff
@@ -0,0 +1,71 @@
+/*Additional queries*/
+/*Get meals that has a price smaller than a specific price fx 90*/
+SELECT
+  *
+FROM
+  meal
+WHERE
+  price < 120
+LIMIT
+  100;
```
Comment (Elia-Git) [inline databases/week3/additional_queries.sql:10]: Remove lines 9 & 10 , why we should limit to 100 meals...? we should test all meals if they fulfill above condition

File: databases/week3/additional_queries.sql [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  /*Get meals that still has available reservations*/
+SELECT
+  *
+FROM
+  meal
+  JOIN reservation ON reservation.meal_id = meal.id
+LIMIT
+  100;
```
Comment (Elia-Git) [inline databases/week3/additional_queries.sql:18]: Remove , no need to find for  a limited nr. of meals

**Review by Elia-Git — CHANGES_REQUESTED** [review]
Few comments for a qiuck fix

File: databases/week3/queries.sql (hunk 1 above)
Comment (27ManishaPatel) [inline databases/week3/queries.sql]: Good Database !

File: databases/week3/queries.sql [hunk 4]
```diff
@@ -1,30 +1,67 @@
...
+FROM
+  meal
+where
+  id = 1
+  /*Update
+            a meal with any id,
+            fx 1.
+          Update
+            any attribute fx the title
+            or multiple attributes*/
```
Comment (27ManishaPatel) [inline databases/week3/queries.sql]: I prefer to write comment in 1 or 2 line instead of 6 lines 😊

File: databases/week3/queries.sql [hunk 5]
```diff
@@ -28,40 +28,138 @@ VALUES(
...
-            or multiple attributes*/
+  /*Update a meal with any id, fx 1.*/
 UPDATE
   meal
 SET
   description = "tasty belgium fries"
 WHERE
   id = 1
-  /*Delete a meal with any id,
-      fx 1*/
```
Comment (27ManishaPatel) [inline databases/week3/queries.sql]: This comment should not be here Right ?

File: databases/week3/additional_queries.sql [hunk 6]
```diff
@@ -0,0 +1,37 @@
+/*Additional queries*/
+/*Get meals that has a price smaller than a specific price fx 90*/
+SELECT
+  *
+FROM
+  meal
+WHERE
+  price < 120
+LIMIT
```
Comment (27ManishaPatel) [inline databases/week3/additional_queries.sql:9]: I think No need to use LIMIT here

File: databases/week3/additional_queries.sql [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+  title like '%fri%'
+  /*Get meals that has been created between two dates*/
+SELECT
+  *
+FROM
+  meal
+WHERE
+  created_date BETWEEN '2021-08-10'
+  AND '2022-10-01';
+  /*Get only specific number of meals fx return only 5 meals*/
```
Comment (27ManishaPatel) [inline databases/week3/additional_queries.sql:34]: good 👍

File: databases/week3/additional_queries.sql [hunk 8]
```diff
@@ -32,6 +32,40 @@ WHERE
...
+SELECT
+  meal.title,
+  AVG(review.stars)
+FROM
+  meal
+  JOIN review ON review.meal_id = meal.id
+GROUP BY
+  meal.title
+ORDER by
+  AVG(review.stars) ASC
```
Comment (27ManishaPatel) [inline databases/week3/additional_queries.sql:71]: great!

**Review by 27ManishaPatel — APPROVED** [review]
well done!

**Pull request merged**